projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -380,86 +372,94 @@ static void validateType(JsonNode value, Variable variable)
       switch (expectedType) {
          case BOOLEAN:
             if (!value.isBoolean()) {
-               throw new BatfishException("");
+               throw new BatfishException(
+                     String.format("Value: %s, is not a valid Boolean " +
+                           "value.", value.toString()));
             }
             break;
          case COMPARATOR:
             if (!(COMPARATORS.contains(value.textValue()))) {
-               throw new BatfishException("");
+               throw new BatfishException(
+                     String.format("Value: %s, is not a valid comparator.",
+                           value.toString()));
```

Comment:
Since the list of known comparators may not be obvious to the reader, we should print it.

"[%s] is not a known comparator. Valid options are %s." // value then list.

Why do you use `toString` here instead of `textValue`?